src/app/Layouts/defaultPageWrapper.jsx

```diff
@@ -1,23 +1,40 @@
-import React from 'react';
+import React, { Fragment, useContext } from 'react';
 import { node, string, bool } from 'prop-types';
 import HeaderContainer from '../containers/Header';
 import FooterContainer from '../containers/Footer';
 import ManifestContainer from '../containers/Manifest';
 import ServiceWorkerContainer from '../containers/ServiceWorker';
-import { ServiceContextProvider } from '../contexts/ServiceContext';
+import {
+  ServiceContextProvider,
+  ServiceContext,
+} from '../contexts/ServiceContext';
 import { RequestContextProvider } from '../contexts/RequestContext';
+import getOriginContext from '../contexts/RequestContext/getOriginContext';
 import ConsentBanner from '../containers/ConsentBanner';
 import GlobalStyle from '../lib/globalStyles';
 
-const PageWrapper = ({ bbcOrigin, children, id, service, isAmp, pageType }) => (
-  <ServiceContextProvider service={service}>
-    <GlobalStyle />
+const PageWithRequestContext = ({
+  bbcOrigin,
+  children,
+  data,
+  env,
+  id,
+  isAmp,
+  pageType,
+  service,
+}) => {
+  const { lang } = useContext(ServiceContext);
+
+  return (
     <RequestContextProvider
       bbcOrigin={bbcOrigin}
+      data={data}
+      env={env}
```

Comment:
Not very familiar with this code, but where/how is the new `env` prop used by `RequestContextProvider`? I can't see it.

Reply:
I think I left this in accidentally after merge, removing it now